packages/clients/meldemichel/src/index.html

```diff
@@ -34,6 +36,20 @@
           stage ? 'stage.' : ''
         }afm.hamburg.de/intelliform/forms/mml_melde_michel/standard/mml_melde_michel/index`,
         reportServiceId: stage ? '6061' : '6059',
+        /*
+        configOverride: {
+          pins: {
+            initial: {
+              centerOn: true,
+              coordinates,
+            },
+            movable: movablePin,
+            style: {
+              fill: movablePin ? '#0392cf' : '#ff0019',
+            },
+          },
+        },
+        */
```

Comment:
I figured [this](https://github.com/Dataport/polar/pull/2/commits/a3cc780e45b891bffd45cc20bef974aea97f2422) would be a good idea. Do you agree?